src/chatdDb.h

```diff
@@ -284,7 +284,36 @@ class ChatdSqliteDb: public chatd::DbInterface
         if (idx != CHATD_IDX_INVALID)
             stmt << idx;
         stmt.stepMustHaveData("get peer msg count");
-        return stmt.intCol(0);
+        int32_t unReadCount = stmt.intCol(0);
+
+        sql = "select data from history where (chatid = ?1)"
+                "and (userid != ?2)"
+                "and not (updated != 0 and length(data) = 0)"
```

Comment:
Are `CallEnd` messages editable or even deletable? I don't think so, so we can skip those conditions too.